### ISSUE TITLE
Sending files via group actions does not work

### ISSUE THREAD

**Issue by AntonAndreevichMoroz** [Ylianst/MeshCentral#3259]
Sending files does not work if you use group actions. Nothing just happens. If you send a file to each machine separately through the Files tab, then it works.

**Comment by K-O-K** [comment]
Hi,

I just tested this with several combinations of devices, Devices that belongs to different DeviceGroups and selecting Windows and Linux Devices at the same time. All seems to work fine for me, the file gets uploaded, it even created the folder if it didn't existed already.

Did you checked the Events Tab on the Device to see if it says something like this:
![image](https://user-images.githubusercontent.com/25281984/140627386-a6921d52-d29e-4d56-b3f1-6bd61616d570.png)

**Comment by K-O-K** [comment]
Hi,

I found the doing a quick google search:

>Nginx Server

>Nginx server settings can be modified inside the file nginx.conf. Open this file and check for the directive client_max_body_size. Then, change the value (in megabytes) to your maximum file size preference.

>If you do not see this directive in nginx.conf, you can add it to the end of a server, location, or http block like so:

 
```

server {
          ...
          client_max_body_size 64M;
}
```

>This allows for a 64-megabyte upload. Set this number to your preference, save the file, then reload Nginx for the change to take effect.

Credits to: [https://blog.hubspot.com/website/413-request-entity-too-large](https://blog.hubspot.com/website/413-request-entity-too-large)

**Comment by AntonAndreevichMoroz** [comment]
@Ylianst what else can I provide for analysis why mesh crashes with a critical error when sending any file in bulk?

**Comment by Ylianst** [comment]
Looking into this now!!

**Comment by Ylianst** [comment]
Just fixed it, will be in MeshCentral v0.9.52. If you are in a hurry to fix this, edit `/node_modules/meshcentral/webserver.js` search for this line:

```
var ugrp = parent.userGroups[i];
```

Replace with this line:

```
var ugrp = obj.userGroups[i];
```

Then, restart the server, that will fix it.

**Comment by AntonAndreevichMoroz** [comment]
Thanks. It works
